ocs_ci/deployment/deployment.py

```
@@ -4637,6 +4638,95 @@ def add_cacert_ramen_configmap(self):
+                    f"(namespace: {constants.OPENSHIFT_DR_SYSTEM_NAMESPACE})"
+                )
+                dr_hub_csvs = get_csvs_start_with_prefix(
+                    constants.ACM_ODR_HUB_OPERATOR_RESOURCE,
+                    constants.OPENSHIFT_DR_SYSTEM_NAMESPACE,
+                )
+                if not dr_hub_csvs:
+                    raise UnexpectedBehaviour(
+                        f"No CSV starting with '{constants.ACM_ODR_HUB_OPERATOR_RESOURCE}' "
+                        f"found in {constants.OPENSHIFT_DR_SYSTEM_NAMESPACE}. "
+                        f"The DR hub operator may not have been deployed yet."
+                    )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)